db/seeds.rb

```diff
@@ -122,7 +122,14 @@
   monthly_medical_expenses: 200,
   monthly_court_ordered_expenses: nil,
   care_expenses: ["childcare"],
-  medical_expenses: ["health_insurance", "prescriptions"],
+  medical_expenses: [
```

Comment:
I thought it was cool the way you added an inclusion validation on `member` for a collection last time we did this -- was it for income types? I can't remember. 

Anyway, what do you think about doing that? And/or adding a constant with all possible values?

Without those types of validations I worry that our seed data will get out of sync with our real data